pkgs/build-support/rust/rustc-wrapper/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, runCommand, rustc, sysroot ? null }:
+
+runCommand "${rustc.pname}-wrapper-${rustc.version}" {
+  preferLocalBuild = true;
+  strictDeps = true;
+  inherit (rustc) outputs;
+
+  env = {
+    prog = "${rustc}/bin/rustc";
+    sysroot = lib.optionalString (sysroot != null) "--sysroot ${sysroot}";
+  };
+
+  passthru = {
+    inherit (rustc) pname version src llvm llvmPackages;
+    unwrapped = rustc;
```

Comment:
BTW thank you for calling this `unwrapped` instead of `rustc`.

The `stdenv.cc.cc` naming is probably a mistake, but it's used everywhere :frown: